### DISCUSSION TITLE
Why the property name of Hanging Punctuation is Height?

### DISCUSSION THREAD

**Discussion by lindexi** [OfficeDev/Open-XML-SDK#1216] [Q&A]
I'm very curious why this property name is called `Height`, and the actual meaning is `Hanging Punctuation`.

```csharp
[DocumentFormat.OpenXml.SchemaAttr(0, "hangingPunct")]
public DocumentFormat.OpenXml.BooleanValue Height { get; set; }
```

See https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.drawing.textparagraphpropertiestype.height?view=openxml-2.8.1

Thank you.

**Comment by mikeebowen** [comment]
It comes from a printing term from hanging punctuation.
```
As hanging punctuation alleviates the disruption from small cap height marks, 
it follows that you should generally hang small cap height marks such as quotation marks (” “ ’ ‘) and hyphens (– —). 
You should avoid hanging full cap height punctuation marks such as the exclamation point (!) and question mark (?).
```
https://webdesign.tutsplus.com/articles/getting-the-hang-of-hanging-punctuation--cms-19890

**Reply by lindexi** [reply]
Thank you @mikeebowen
